### PR TITLE
My first post about the Vert.X-Swagger project

### PR DESCRIPTION
**Pull request by phiz71**
Hi.
Would it be possible to see what the blog post looks like before publishing it ?
Because embedded scripts and custom tags (NOTE, INFO, etc) are not well rendered on github.
Thanks

### CODE REVIEW

**Comment by alexlehm** [comment]
you should be able to create the site locally

**Comment by phiz71** [comment]
Thanks, it works ! 👍 
And now I can see that I must commit some corrections before publishing

**Comment by phiz71** [comment]
Now it's ok for me. I did a lot of refactoring to get a good rendering.

File: src/site/blog/posts/2017-05-12-vertx-swagger-presentation.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+---
+title: Presentation of the Vert.X - Swagger project
```
Comment (vietj) [inline src/site/blog/posts/2017-05-12-vertx-swagger-presentation.md]: needs to be _Eclipse Vert.x_

File: src/site/blog/posts/2017-05-12-vertx-swagger-presentation.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+template: post.html
+date: 2017-05-12
+author: phiz71
+draft: true
+---
+
+An introduction to the [Vert.X-Swagger] project. How to use the Swagger-Codegen plugin and the SwaggerRouter class.
+
+## Eclipse Vert.X & Swagger
+[Eclipse Vert.X][vertx-core] and [Vert.X-Web] are very convenient to write REST API and especially the [Router] which is very useful to manage all resources of an API.
```
Comment (vietj) [inline src/site/blog/posts/2017-05-12-vertx-swagger-presentation.md]: Can just be _Vert.x_, at _Eclipse Vert.x_ is used in the title above

**Comment by vietj** [comment]
a couple of corrections, looks good, should be published early next week!

File: src/site/blog/posts/2017-05-12-vertx-swagger-presentation.md (hunk 1 above)
Comment (phiz71) [inline src/site/blog/posts/2017-05-12-vertx-swagger-presentation.md]: except that my project's name is vertx-swagger.
And if I remove the spaces ^^ ?  _Vert.X-Swagger_

File: src/site/blog/posts/2017-05-12-vertx-swagger-presentation.md (hunk 2 above)
Comment (phiz71) [inline src/site/blog/posts/2017-05-12-vertx-swagger-presentation.md]: ok

**Comment by cescoffier** [comment]
The post has been published, thanks !

**Pull request merged**